### PR TITLE
Added keyframes feature to monolithic mode

### PR DESCRIPTION
**Pull request by tiagojsalmeida**
Would be great to have animations working since we use CXS live and it's affecting our animations.
I only worked on Monolithic mode at the moment.

Related with https://github.com/jxnblk/cxs/issues/46

### CODE REVIEW

File: src/monolithic/index.js [hunk 1]
```diff
@@ -71,6 +71,18 @@ const parse = (selector, styles, media) => {
       parse(selector + key, value, media)
         .forEach(r => rules.push(r))
       continue
+    } else if (/^animationName/.test(key)) {
```
Comment (jxnblk) [inline src/monolithic/index.js]: Why use a custom naming convention rather than using `@keyframes`?

File: src/monolithic/index.js [hunk 2]
```diff
@@ -99,6 +126,9 @@ const createRule = (selector, decs, media) => {
   return css
 }
 
+const createAnimationRule = (selector, decs) =>
+  `${selector}{${decs.join(' ')}}`;
```
Comment (jxnblk) [inline src/monolithic/index.js]: I think this needs to be `.join(';')` to properly separate declarations

**Comment by richardgorman** [comment]
@jxnblk Can this request be merged?

**Comment by jxnblk** [comment]
Closing due to inactivity

**Pull request closed without merging**